src/erlcloud_application_autoscaler.erl

```diff
@@ -61,10 +61,14 @@
 -type response_attribute() :: string() | integer().
 -type response_key() :: atom().
 -type response() :: [{response_key(), response_attribute()}].
+-type error_type() :: binary().
+-type error_message() :: binary().
 -type ok_error_response() :: {ok, jsx:json_term()}
                            | {error, metadata_not_available
                                    | container_credentials_unavailable
-                                   | erlcloud_aws:httpc_result_error()}.
+                                   | erlcloud_aws:httpc_result_error()}
+                           | {error, {error_type(), error_message()}}
```

Comment:
Please use the pattern here, not this redundant thing.

Reply:
Fixed.